helpers/constants.js

```diff
@@ -110,7 +110,10 @@ var constants = {
 		offset: 1451520, // Start rewards at block (n)
 		distance: 3000000, // Distance between each milestone
 	},
-	// WARNING: When changing totalAmount you also need to change getBlockRewards(int) SQL function!
+	// Every version starts on provided height
+	blockVersions: [
```

Comment:
Is this replacing or affecting somehow the exceptions to freeze Dapp transfers?
Could we use it to remove those exceptions?